chia/wallet/wallet_node.py

```diff
@@ -1392,6 +1395,8 @@ async def validate_block_inclusion(
             start, end, peer_request_cache, all_peers
         )
         if blocks is None:
+            if self._shut_down:
+                self.log.debug(f"shutdown, dont fetch blocks {start} {end}")
             self.log.error(f"Error fetching blocks {start} {end}")
```

Comment:
we should have an else here, just to make it clear.
instead of the current message, we should probably put `"Shutting down, block fetching from: {start}  to {end} canceled."` just to make it easier for users to understand